diff --git a/paragraphs.field_widget.inc b/paragraphs.field_widget.inc
index a2dd4e5..1784492 100644
--- a/paragraphs.field_widget.inc
+++ b/paragraphs.field_widget.inc
@@ -145,6 +145,7 @@ function paragraphs_field_multiple_value_form($field, $instance, $langcode, $ite
       '#suffix' => '</div>',
       '#max_delta' => $max,
       '#instance' => $instance,
+      '#element_validate' => array('_paragraph_multiple_field_validate'),
     );
 
     // Add 'add more' button, if not working with a programmed form.
@@ -1084,3 +1085,13 @@ function paragraphs_restore_submit($form, &$form_state) {
 
   $form_state['rebuild'] = TRUE;
 }
+
+/**
+ * Form Validation for submitting a node with required paragraph fields
+ *
+ */
+function _paragraph_multiple_field_validate($element, &$form_state) {
+  if (($element['#required'] == 1 && !isset($element[0])) || (isset($form_state['field'][$element['#field_name']][LANGUAGE_NONE]['real_items_count']) && $form_state['field'][$element['#field_name']][LANGUAGE_NONE]['real_items_count'] == 0)) {
+    form_error($element, t('The "!name" field is required', array('!name' => t($element['#title']))));
+  }
+}
\ No newline at end of file
